dkg/dkg.go

```diff
@@ -181,7 +182,22 @@ func Run(ctx context.Context, conf Config) (err error) {
 		}
 		peerMap[p.ID] = nodeIdx
 	}
-	tp := newFrostP2P(tcpNode, peerMap, key, def.Threshold, def.NumValidators)
+
+	bf := bcast.New(tcpNode, peerIds, key)
+
+	// register bcast callbacks for frostp2p
+	tp, frostCallback := newFrostP2P(tcpNode, peerMap, bf.BroadcastFunc, def.Threshold, def.NumValidators)
```

Comment:
why not pass the whole bcast component into `newFrostP2P` and register callbacks inside that function?